test/integration/global-network-firewall-policy/global_firewall_policy_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package global_network_firewall_policy
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGlobalNetworkFirewallPolicy(t *testing.T) {
+	fwp := tft.NewTFBlueprintTest(t)
+	fwp.DefineVerify(
+		func(assert *assert.Assertions) {
+			fwp.DefaultVerify(assert)
+			projectID := fwp.GetStringOutput("project_id")
+			policyName := fwp.GetStringOutput("firewall_policy_name")
+
+			policy := gcloud.Run(t, fmt.Sprintf("compute network-firewall-policies describe %s --global --project %s", policyName, projectID))
```

Comment:
nit: you can use format strings with gcloud.Runf
```suggestion
			policy := gcloud.Run(t,"compute network-firewall-policies describe %s --global --project %s", policyName, projectID)
```